migrations/20200714113210-create-user.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+module.exports = {
+    up: async (queryInterface, Sequelize) => {
+        await queryInterface.createTable('Users', {
+            id: {
+                allowNull: false,
+                autoIncrement: true,
+                primaryKey: true,
+                type: Sequelize.INTEGER,
+            },
+            username: {
+                type: Sequelize.STRING,
+            },
+            discordId: {
+                type: Sequelize.STRING,
+                allowNull: false,
+                unique: true,
+            },
+            linkedin: {
+                type: Sequelize.STRING,
+                default: '',
+            },
+            github: {
+                type: Sequelize.STRING,
+                default: '',
+            },
+            funfact: {
+                type: Sequelize.STRING,
+                default: '',
+            },
+            pronouns: {
+                type: Sequelize.STRING,
+                default: '',
+            },
+            enrolled: {
+                type: Sequelize.BOOLEAN,
+                default: true,
+            },
+            createdAt: {
+                allowNull: false,
+                type: Sequelize.DATE,
+            },
+            updatedAt: {
```

Comment:
we could probably add a column which indicates if the user is receiving matches or if they paused the service

Reply:
Oh I see there's `enrolled`